spec/cypress/integration/Cookies.feature

```diff
@@ -18,6 +18,9 @@ Feature: Cookie page
     Then cookie preferences have changed
     And "cookie consent radio" with value "off" is checked
 
+    And the page should be accessible
+    And percy should be sent snapshot
```

Comment:
I presume we will add a lot more of that soon?

Reply:
yep!